pom.xml

```diff
@@ -1538,6 +1538,11 @@
                 <artifactId>org.wso2.carbon.identity.application.authenticator.apple</artifactId>
                 <version>${social.authenticator.apple.version}</version>
             </dependency>
+            <dependency>
+                <groupId>org.wso2.carbon.extension.identity.authenticator</groupId>
+                <artifactId>org.wso2.carbon.extension.identity.authenticator.github.connector</artifactId>
+                <version>${social.authenticator.github.version}</version>
+            </dependency>
```

Comment:
```suggestion
```

Reply:
Fixed with https://github.com/wso2/product-is/pull/17970/commits/b26e54e865a3e322fcbcdb4aec0ad8e8eebe8bf1